services/nginx/Dockerfile

```diff
@@ -1,6 +1,11 @@
 FROM nginx
 
 # support running as an arbitrary user which belogs to the root group
-RUN chmod g+rwx /var/cache/nginx /var/run /var/log/nginx
+RUN chmod g+rwx /var/cache/nginx /var/run /var/log/nginx && \
+    apt-get update && \
+    apt-get install --no-install-recommends --no-install-suggests -y python3-jinja2 python3-yaml python3-minimal && \
+    apt-get remove --purge --auto-remove -y && rm -rf /var/lib/apt/lists/*
```

Comment:
I think we may need to tweak the permissions of `/etc/nginx/api_conf.d/` in order for `build-backends.py` to be able to write files to this in OpenShift. It works locally, but I had to allow write access in order for the deployment to not complain about permissions at this point: https://github.com/RedHatInsights/turnpike/pull/3/files#diff-55e8767d4db8bc18f41d3ce736c3f037R37

Reply:
Yep, makes sense. It sounds like we need `api_conf.d` to be `0775`?

Reply:
Yep! That's what got it to work for me.